turbo.json

```diff
@@ -461,6 +461,7 @@
     "NEXT_PUBLIC_POSTHOG_KEY",
     "NEXT_PUBLIC_POSTHOG_HOST",
     "VAPID_PRIVATE_KEY",
-    "HUDDLE01_API_TOKEN"
+    "HUDDLE01_API_TOKEN",
+    "REPLEXICA_API_KEY"
```

Comment:
added to fix 
![Screenshot 2024-11-20 at 1 40 13 PM](https://github.com/user-attachments/assets/efd5f8ca-4ef3-4e5f-b7a3-ad80469d8476)
`171:34 error $REPLEXICA_API_KEY is not listed as a dependency in turbo.json turbo/no-undeclared-env-vars`